recipes/genie/all/conanfile.py

```diff
@@ -66,17 +74,15 @@ def build(self):
                     cxx = "clang++" if self.settings.compiler == "clang" else "g++"
             self._patch_compiler(cc, cxx)
 
-            make = tools.get_env("CONAN_MAKE_PROGRAM", tools.which("make") or tools.which("mingw32-make"))
-            if not make:
-                raise ConanInvalidConfiguration("This package needs 'make' in the path to build")
-
+            autotools = AutoToolsBuildEnvironment(self, win_bash=tools.os_info.is_windows)
             with tools.chdir(self._source_subfolder):
-                self.run(make, win_bash=tools.os_info.is_windows)
+                autotools.make(args=["OS={}".format(self._os), "config={}".format(self._genie_config)])
 
     def package(self):
         self.copy("LICENSE", dst="licenses", src=self._source_subfolder)
         bin_ext = ".exe" if self.settings.os == "Windows" else ""
         self.copy("genie{}".format(bin_ext), dst="bin", src=os.path.join(self._source_subfolder, "bin", self._os))
+        self.copy("*.lua", dst="res", src=os.path.join(self._source_subfolder, "src"))
```

Comment:
I copied them just to be safe.
```suggestion
        if self.settings.build_type == "Debug":
            self.copy("*.lua", dst="res", src=os.path.join(self._source_subfolder, "src"))
```